src/Appwrite/Platform/Modules/Proxy/Http/Rules/API/Create.php

```diff
@@ -60,7 +61,7 @@ public function __construct()
             ->label('abuse-limit', 10)
             ->label('abuse-key', 'userId:{userId}, url:{url}')
             ->label('abuse-time', 60)
-            ->param('domain', null, new ValidatorDomain(), 'Domain name.')
+            ->param('domain', null, new AnyOf([new ValidatorDomain(), new AppwriteDomain()]), 'Domain name.')
```

Comment:
Should this be AllOf? This will succeed even if `AppwriteDomain` validates as false